src/sass/Fabric.Responsive.Utilities.Output.scss

```diff
@@ -242,781 +242,771 @@
-  }
-  .ms-u-mdPush8 {
-    @include ms-u-mdPush8;
-  }
-  .ms-u-mdPush7 {
-    @include ms-u-mdPush7;
-  }
-  .ms-u-mdPush6 {
-    @include ms-u-mdPush6;
-  }
-  .ms-u-mdPush5 {
-    @include ms-u-mdPush5;
-  }
-  .ms-u-mdPush4 {
-    @include ms-u-mdPush4;
-  }
-  .ms-u-mdPush3 {
-    @include ms-u-mdPush3;
-  }
-  .ms-u-mdPush2 {
-    @include ms-u-mdPush2;
-  }
-  .ms-u-mdPush1 {
-    @include ms-u-mdPush1;
-  }
-  .ms-u-mdPush0 {
-    @include ms-u-mdPush0;
-  }
-  
-  // Medium Offset classes 0 ... 11
-  .ms-u-mdOffset11 {
-    margin-left: 91.66666666666666%;
-  }
-  .ms-u-mdOffset10 {
-    margin-left: 83.33333333333334%;
-  }
-  .ms-u-mdOffset9 {
-    margin-left: 75%;
-  }
-  .ms-u-mdOffset8 {
-    margin-left: 66.66666666666666%;
-  }
-  .ms-u-mdOffset7 {
-    margin-left: 58.333333333333336%;
-  }
-  .ms-u-mdOffset6 {
-    margin-left: 50%;
-  }
-  .ms-u-mdOffset5 {
-    margin-left: 41.66666666666667%;
-  }
-  .ms-u-mdOffset4 {
-    margin-left: 33.33333333333333%;
-  }
-  .ms-u-mdOffset3 {
-    margin-left: 25%;
-  }
-  .ms-u-mdOffset2 {
-    margin-left: 16.666666666666664%;
-  }
-  .ms-u-mdOffset1 {
-    margin-left: 8.333333333333332%;
-  }
-  .ms-u-mdOffset0 {
-    margin-left: 0;
-  }
+.ms-u-md12 {
+  @include ms-u-md12;
+}
+.ms-u-md11 {
+  @include ms-u-md11;
+}
+.ms-u-md10 {
+  @include ms-u-md10;
+}
+.ms-u-md9 {
+  @include ms-u-md9;
+}
+.ms-u-md8 {
+  @include ms-u-md8;
+}
+.ms-u-md7 {
+  @include ms-u-md7;
+}
+.ms-u-md6 {
+  @include ms-u-md6;
+}
+.ms-u-md5 {
+  @include ms-u-md5;
+}
+.ms-u-md4 {
+  @include ms-u-md4;
+}
+.ms-u-md3 {
+  @include ms-u-md3;
+}
+.ms-u-md2 {
+  @include ms-u-md2;
+}
+.ms-u-md1 {
+  @include ms-u-md1;
+}
+.ms-u-mdPull12 {
+  @include ms-u-mdPull12;
+}
+.ms-u-mdPull11 {
+  @include ms-u-mdPull11;
+}
+.ms-u-mdPull10 {
+  @include ms-u-mdPull10;
+}
+.ms-u-mdPull9 {
+  @include ms-u-mdPull9;
+}
+.ms-u-mdPull8 {
+  @include ms-u-mdPull8;
+}
+.ms-u-mdPull7 {
+  @include ms-u-mdPull7;
+}
+.ms-u-mdPull6 {
+  @include ms-u-mdPull6;
+}
+.ms-u-mdPull5 {
+  @include ms-u-mdPull5;
+}
+.ms-u-mdPull4 {
+  @include ms-u-mdPull4;
+}
+.ms-u-mdPull3 {
+  @include ms-u-mdPull3;
+}
+.ms-u-mdPull2 {
+  @include ms-u-mdPull2;
+}
+.ms-u-mdPull1 {
+  @include ms-u-mdPull1;
+}
+.ms-u-mdPull0 {
+  @include ms-u-mdPull0;
+}
+.ms-u-mdPush12 {
+  @include ms-u-mdPush12;
+}
+.ms-u-mdPush11 {
+  @include ms-u-mdPush11;
+}
+.ms-u-mdPush10 {
+  @include ms-u-mdPush10;
+}
+.ms-u-mdPush9 {
+  @include ms-u-mdPush9;
+}
+.ms-u-mdPush8 {
+  @include ms-u-mdPush8;
+}
+.ms-u-mdPush7 {
+  @include ms-u-mdPush7;
+}
+.ms-u-mdPush6 {
+  @include ms-u-mdPush6;
+}
+.ms-u-mdPush5 {
+  @include ms-u-mdPush5;
+}
+.ms-u-mdPush4 {
+  @include ms-u-mdPush4;
+}
+.ms-u-mdPush3 {
+  @include ms-u-mdPush3;
+}
+.ms-u-mdPush2 {
+  @include ms-u-mdPush2;
+}
+.ms-u-mdPush1 {
+  @include ms-u-mdPush1;
+}
+.ms-u-mdPush0 {
+  @include ms-u-mdPush0;
+}
+
+// Medium Offset classes 0 ... 11
+.ms-u-mdOffset11 {
+  margin-left: 91.66666666666666%;
```

Comment:
Is there a reason there aren't mixins for these classes as well? If we're removing the higher-order media queries, these should be accounted for as well.